pkg/controller/directimagemigration/task.go

```diff
@@ -277,3 +280,25 @@ func (t *Task) getDestinationClient() (compat.Client, error) {
 	}
 	return client, nil
 }
+
+// Get the extended phase description for a phase.
+func (t *Task) getPhaseDescription(phaseName string) string {
+	// Log the extended description of current phase
+	if phaseDescription, found := PhaseDescriptions[t.Phase]; found {
+		return phaseDescription
+	}
+	t.Log.V(4).Info("Missing phase description for phase: " + phaseName)
+	// If no description available, just return phase name.
+	return phaseName
+}
+
+// Log the "[RUN] <Phase description>" phase kickoff string unless
+// DISMs are already logging a duplicate phase description.
+// This is meant to cut down on log noise when two controllers
+// are waiting on the same thing.
+func (t *Task) logRunHeader() {
+	if t.Phase != WaitingForDirectImageStreamMigrationsToComplete {
```

Comment:
Is there some way we tell the user that we are still waiting on completion?

Reply:
Yeah, the DVM controller or DIM controller will be logging a message every 3 seconds stating that they're waiting. This is just meant to reduce the log clutter (otherwise both controllers write the same log message every 3 seconds)